pkg/cluster/meta/topology_dm_test.go

```diff
@@ -205,6 +205,29 @@ dm-worker_servers:
   - host: 172.16.5.138
 `), &topo)
 	c.Assert(err, NotNil)
-	c.Assert(err.Error(), Equals, "platform mismatch for '172.16.5.138' as in 'dm-master_servers:darwin/arm64' and 'dm-worker_servers:linux/arm64'")
+	c.Assert(err.Error(), Equals, "platform mismatch for '172.16.5.138' between 'dm-master_servers:darwin/arm64' and 'dm-worker_servers:linux/arm64'")
 
 }
+
+func (s *metaSuiteDM) TestCountDir(c *C) {
+	topo := DMTopologySpecification{}
+
+	err := yaml.Unmarshal([]byte(`
+global:
+  user: "test1"
+  ssh_port: 220
+  deploy_dir: "test-deploy"
+dm-master_servers:
+  - host: 172.16.5.138
+    deploy_dir: "master-deploy"
+    data_dir: "/test-data/data-1"
+dm-worker_servers:
+  - host: 172.16.5.53
+    data_dir: "test-1"
+`), &topo)
```

Comment:
Test cases not enough, the following cases should be covered:

1. absolute global data path + relative instance  data directory
2. relative global data directory + absolute instance  data directory
3. absolute global data directory + absolute instance data directory
4. relative global data directory + relative instance data directory